src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -59,9 +57,7 @@ See the [Amazon Cognito documentation]( https://docs.aws.amazon.com/cognito/late
 
 ### Set up and configure your Auth category
```

Comment:
maybe we should change the title "Set up and configure Amplify Auth"

Reply:
Like it - more consistent on expectations with the page title too.